core/overlays/stage/configmaps.yaml

```diff
@@ -23,7 +23,7 @@ data:
   postgresql-host: pgbouncer.thoth-graph-stage.svc
   rsyslog-host: ""
   rsyslog-port: ""
-  storage-bucket-name: ocp-prod-stage
+  storage-bucket-name: ocp-stage
```

Comment:
why do we not use `ocp-prod-stage` here?

Reply:
either one is fine for me, we should just stick with one.